shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -695,7 +695,7 @@ module DataFlowMake<LocationSig Location, InputSig<Location> Lang> {
    * Constructs a global data flow computation.
    */
   module Global<ConfigSig Config> implements GlobalFlowSig {
-    private module C implements FullStateConfigSig {
+    private module StateConfig implements FullStateConfigSig {
```

Comment:
Please revert. It's a short-scoped private name, and making it longer may excessively bloat the length of DIL/RA names.

Reply:
Hmm, I was not aware that this was a concern! Thx!